preprocessors/line-charts/charts.py

```diff
@@ -124,7 +125,7 @@ def get_chart_info():
         return jsonify("Invalid Preprocessor JSON format"), 500
 
     logging.debug("Sending response")
-    # print(data)
+    logging.debug(data)
```

Comment:
@jaydeepsingh25 Not sure we want to dump all of the actual data into logs.

Reply:
I tested with the [etherscan.io chart that contains many points](https://etherscan.io/chart/etherprice), and the single log line from line-chart is 438K alone, amking it very difficult to watch the overall logs:

`-rw-rw-r-- 1 jeffbl jeffbl 438K Aug  7 14:53 /home/jeffbl/foo.log`

Plus if it outputs any of the title, axes, etc. then this is end-user content we should not be storing in logs.

Can we just summarize it with something like the number of points, total size, or some other proxy value to give an idea of the response?